src/Fetch.ml

```diff
@@ -379,6 +379,29 @@ module Response = struct
   external clone : t = "clone" [@@bs.send.pipe: t]
 end
 
+(* This belongs to XHR spec and will probably be moved to another repo in the future *)
+
+module Blob = struct
+  type t
+end
+
+module FormData = struct 
+  type t = formData
+
+  external make : unit -> t = "FormData" [@@bs.new]
+
+  (* Added for React Native polyfill compatibility *)
+  external appendObject : string -> < .. > Js.t -> unit = "append" [@@bs.send.pipe : t]
```

Comment:
I think this needs to be restricted to a specific shape, given this comment: https://github.com/reasonml-community/bs-fetch/pull/14#discussion_r177700547

Reply:
Yeah, it looks like a specific shape, but I have no idea really. I know very little about React Native.